configuration/configuration.go

```diff
@@ -811,6 +812,11 @@ func (c *Registry) GetTogglesServiceURL() string {
 	return c.v.GetString(varTogglesServiceURL)
 }
 
+// GetSentryDSN returns the DSN needed to authenticate and communicate with Sentry service
+func (c *Registry) GetSentryDSN() string {
```

Comment:
No need to pass DSN via configuration. raven-go will automatically load it from the SENTRY_DSN env var.
And this GetSentryDSN() function won't work anyway. It will look for F8_SENTRY_DSN env var. Not SENTRY_DSN.

Reply:
done